language/fr/contact.lang

```diff
@@ -0,0 +1,11 @@
+<?php
+
+include("{$_EnginePath}language/en/contact.lang");
```

Comment:
This file's translation is completed, therefore you can remove this default import.

Reply:
Done.